recipe/meta.yaml

```
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
```

```suggestion
  number: 0
```
As it's a first build